Pyrado/scripts/training/qq-su_ppo_sprl_multiple.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) 2020, Fabio Muratore, Honda Research Institute Europe GmbH, and
```

Comment:
please only push **1** SPRL training file per env and subroutine, and exlude all others